web/src/search.js

```diff
@@ -90,7 +90,8 @@ export function initialize() {
             search_map = suggestions.lookup_table;
             return suggestions.strings;
         },
-        fixed: true,
+        container: "<div class='typeahead dropdown-menu'></div>",
```

Comment:
This line should be removed... It just overrides the default container with the exact same value.

Rest looks fine to me!

Reply:
Ah of course! Removed now :)